demo/blocks_style/run.py

```diff
@@ -70,7 +70,7 @@
             height="auto",
             container=False,
         )
-        chat = gr.Chatbot("hi", color_map=("pink", "blue")).style(
+        chat = gr.Chatbot([("hi", "good bye")], color_map=("pink", "blue")).style(
             rounded=False,
         )
```

Comment:
Might as well remove this since it's deprecated
```suggestion
        chat = gr.Chatbot([("hi", "good bye")], color_map=("pink", "blue"))
```